rollup.config.js

```diff
@@ -23,6 +23,22 @@ export default [
       commonjs(),
     ],
   },
+  {
+    input: 'src/web3/index.ts',
+    external: ['react', 'prop-types'],
+    output: [
+      { file: 'web3/index.js', format: 'cjs' },
+      { file: 'web3/index.esm.js', format: 'es' },
+    ],
+    plugins: [
+      ts(),
+      resolve(),
+      babel({
+        extensions: ['.ts', '.js', '.tsx', '.jsx'],
```

Comment:
(nit) this is new code, so we probably shouldnt support js file right? unless this is required because we are transpiling es6 code from node_modules or something?

```suggestion
        extensions: ['.ts', '.tsx'],
```